courses/free-courses-en.md

```diff
@@ -666,6 +666,7 @@
 * [Java Server Tutorials - Happy Coding](https://happycoding.io/tutorials/java-server/) - Kevin Workman
 * [Java tutorial for beginners](https://youtube.com/playlist?list=PLZPZq0r_RZOMhCAyywfnYLlrjiVOkdAI1) - Bro Code (YouTube)
 * [Java Tutorial For Beginners](https://youtube.com/playlist?list=PLsyeobzWxl7oZ-fxDYkOToURHhMuWD1BK) - Telusko (YouTube)
+* [Java Tutorial for Beginners | Full Course](https://www.youtube.com/watch?v=8cm1x4bC610) - Navin Reddy, Telusko
```

Comment:
There is a lot of video overlap between previous entry and this entry as it is from the same creator but looks like lot of content is updated from the previous one. So this one would be a value add

Reply:
Thank you @shrn01

Reply:
we prefer the playlist